traduki/sqla.py

```diff
@@ -172,12 +169,31 @@ def i18n_relation(column=None, comparator_factory=TranslationComparator,
         """
         if column is not None:
             if 'primaryjoin' in kwargs:
-                raise ArgumentError(
-                    "You cannot supply 'primaryjoin' argument to 'i18n_relation'")
+                raise ArgumentError("You cannot supply 'primaryjoin' argument to 'i18n_relation'")
             kwargs['primaryjoin'] = column == Translation.id
 
-        return relationship(
-            Translation, comparator_factory=comparator_factory,
-            extension=extension, lazy=lazy, **kwargs)
+        res = relationship(
+            Translation,
+            comparator_factory=comparator_factory,
+            lazy=lazy,
+            # This required only for sqlalchemy < 1.3.4. See https://github.com/sqlalchemy/sqlalchemy/issues/4691
+            active_history=True,
+            **kwargs
+        )
+
+        # We must attach the event on before_configured, otherwise it is too late if we use after_configured, as other
```

Comment:
This copy is a bit awkward, maybe change it to something like _We must attach the event on before_configured. If we use after_configured, we're too late as other hooks will already be installed by SQLAlchemy._

Reply:
done